site2/docs/admin-api-brokers.md

```diff
@@ -147,7 +147,7 @@ admin.brokers().getOwnedNamespaces(cluster,brokerUrl);
 
 ### Dynamic broker configuration
 
-One way to configure a Pulsar [broker](reference-terminology.md#broker) is to supply a [configuration](reference-configuration.md#broker) when the broker is [started up](reference-cli-tools.md#pulsar-broker).
+One way to configure a Pulsar [broker](reference-terminology.md#broker) is to supply a [configuration](reference-configuration.md#broker) when the broker is [started up](reference-cli-tools.md).
```

Comment:
```suggestion
One way to configure a Pulsar [broker](reference-terminology.md#broker) is to supply a [configuration](reference-configuration.md#broker) when the broker is [started up](reference-cli-broker-tool.md).
```

Reply:
Thanks for pointing this out, but I think this PR should focus on fixing the error anchors. We may improve the `reference-cli-tools.md` in another PR.

Reply:
Make sense.